mesonbuild/dependencies/dub.py

```diff
@@ -56,6 +57,8 @@ class FindTargetEntry(TypedDict):
         search: str
         artifactPath: str
 
+DubDescriptionSource = Enum('DubDescriptionSource', ['Local', 'External'])
```

Comment:
Could we use the class form? That tends to work better with tools like mypy than the functional interface does.